py/moq-rs/moq/publish.py

```
@@ -190,6 +190,9 @@ def dynamic(self) -> BroadcastDynamic:
     def publish_media(self, format: str, init: bytes) -> MediaProducer:
         return MediaProducer(self._inner.publish_media(format, init))
 
+    def publish_media_on_track(self, track: TrackProducer, format: str, init: bytes) -> MediaProducer:
```

Ruff (0.15.15)

[error] Function argument `format` is shadowing a Python builtin

(A002)